src/applications/check-in/components/AppointmentDisplay/AppointmentListItem.jsx

```diff
@@ -26,6 +26,21 @@ const AppointmentListItem = props => {
   const selectFeatureToggles = useMemo(makeSelectFeatureToggles, []);
   const { is45MinuteReminderEnabled } = useSelector(selectFeatureToggles);
 
+  const detailsAriaLabel = () => {
```

Comment:
what was the reason to re-factor this?

Reply:
It was in the wire frames the aria label changed.

Reply:
https://www.figma.com/file/sK2vIx6giAynNqUFR0l4Wx/Production-%7C-Check-in?type=design&node-id=29-614&mode=design&t=affs9Cf0bystQVWb-0

Reply:
oh nice, thanks for the link to that view!